### ISSUE TITLE
Log regression after migration from log15

### ISSUE THREAD

**Issue by drTr0jan** [grafana/grafana#45752]
**What happened**:
There're two regressions after #41636:
1. All syslog messages are sent as `Emergency` severity.
2. Changed log fields (`logger`, `t`, `lvl`, `msg`...) order breaks external integrations, ex. [fail2ban](https://github.com/fail2ban/fail2ban/blob/master/config/filter.d/grafana.conf).

**What you expected to happen**:
1. All syslog messages are consistent to [level package](https://godoc.org/github.com/go-kit/log/level) messages priority.
2. Log fields order is consistent to log15 fields order (ex.: `t`, `lvl`, `msg`, `logger`, `userId`, `orgId`. `uname`, `error`, `remote_addr`).

**How to reproduce it (as minimally and precisely as possible)**:
1. Install Grafana 8.4.1 and configure `[log]` as `mode = syslog`.
2. Install Grafana 8.4.1 and try to login.

**Environment**:
- Grafana version: 8.4.0-beta1 and newer
- Data source type & version: N/A
- OS Grafana is installed on: FreeBSD
- User OS & Browser: N/A
- Grafana plugins: N/A
- Others:

**Comment by zuchka** [comment]
see this duplicate issue for more detailed logging info re: syslog as emergency severity:

https://github.com/grafana/grafana/issues/45837

**Comment by ying-jeanne** [comment]
The priority of syslog is definitely need to be fixed, but I am not sure to keep the key order of log15, need to take a look how the key is ordered in log15 and whether it makes sense to to produced the ordering with gokit/log
```
Log fields order is consistent to log15 fields order (ex.: t, lvl, msg, logger, userId, orgId. uname, error, remote_addr)
```

**Comment by krzysion** [comment]
Fields are missing from logs: userId, orgId, uname

Grafana 8.3.6:
![2022-03-29 07_31_05-Window](https://user-images.githubusercontent.com/36912975/160545926-bbc882e7-55ea-4355-a562-cae16d20c37a.png)

Grafana 8.4.4:
![2022-03-29 07_24_30-Window](https://user-images.githubusercontent.com/36912975/160545966-06070ca1-a0f0-4a12-b536-98efd7b43f23.png)

**Comment by ying-jeanne** [comment]
checked I think this specific issue is not related to logging migration, but how the context is passed into contexthandler @idafurjes could you take a look?

**Comment by ying-jeanne** [comment]
@krzysion hey Krzysion, the problem you mentioned is not related to this issue, please open a separate ticket. Thanks.

**Comment by nomis** [comment]
Is the severity level going to be fixed soon? There have been multiple releases of 8.4 now that are unusable with syslog.

I'd argue that with a key/value log format it's the other software that is wrong for expecting a particular order...

**Comment by marefr** [comment]
Hopefully able to open a fix for this during this week and hopefully include in 8.4 patch release next week.
